draft-nro-bulk-rdap.md

```diff
@@ -48,184 +48,294 @@ feature of this service.
+      "handle": "AAAA-ARIN",
+      "roles": [ "administrative", ... ]
       "links":
       [
         {
-          "value": "https://example.net/ip/2001:db8:1::/48",
+          "value": "https://example.net/ip/2001:db8::/48",
           "rel": "self",
-          "href": "https://example.net/ip/2001:db8:1::/48",
+          "href": "https://example.net/entity/AAAA-ARIN",
           "type": "application/rdap+json"
-        },
-        ...
-      ],
-      "entities":
+        }
+      ]
+    },
+    ...
+  ]
+}
+...
+{
+  "rdapConformance": [ "rdap_level_0", "nro_rdap_profile_0", "nroBulkRdap1", ... ],
+  "objectClassName": "autnum",
+  "handle": "ZZZZ-ARIN",
+  "startAutnum": 65536,
+  "endAutnum": 65541,
+  ...
+  "links":
+  [
+    {
+      "value": "https://example.net/autnum/65536",
+      "rel": "self",
+      "href": "https://example.net/autnum/65536",
+      "type": "application/rdap+json"
+    },
+    ...
+  ],
+  "entities":
+  [
+    {
+      "objectClassName": "entity",
+      "handle": "BBBB-ARIN",
+      "roles": [ "technical", ... ]
+      "links":
       [
         {
-          "objectClassName": "entity",
-          "handle": "BBBB-RIR",
-          "roles": [ "technical", ... ]
-          "links":
-          [
-            {
-              "value": "https://example.net/ip/2001:db8:1::/48",
-              "rel": "self",
-              "href": "https://example.net/entity/BBBB-RIR",
-              "type": "application/rdap+json"
-            }
-          ],
-        },
-        ...
-      ],
+          "value": "https://example.net/autnum/65536",
+          "rel": "self",
+          "href": "https://example.net/entity/BBBB-ARIN",
+          "type": "application/rdap+json"
+        }
+      ]
     },
     ...
   ]
 }
+...
 ```
 
-_Figure 1_
-
 # Extension Identifier
 
-The "nroBulkRdap1" extension identifier (see (#rdap_extensions_registry)) MUST be included in the top-level
-"rdapConformance" member of the JSON object for bulk data, to signal adherence to this specification.
+The "nroBulkRdap1" extension identifier (see (#rdap_extensions_registry)) MUST be included as the value for the
+"extensionId" member in the metadata object of a bulk data response.
 
-It is RECOMMENDED that the extension identifier for the NRO RDAP Profile ("nro_rdap_profile_0") also be included in the
-top-level "rdapConformance" member. But, when in conflict, the Bulk RDAP extension requirements SHOULD supersede the NRO
-RDAP Profile extension requirements, primarily to afford bulk data compactness.
+The "rdapConformance" array for each returned data object MUST include both the "nroBulkRdap1" extension identifier and
+the extension identifier for the
+[NRO RDAP Profile](https://bitbucket.org/nroecg/nro-rdap-profile/raw/v1/nro-rdap-profile.txt) ("nro_rdap_profile_0" as
+of this writing). In the data objects, the Bulk RDAP extension requirements MUST supersede the NRO RDAP Profile
+extension requirements, primarily to afford bulk data compactness.
```

Comment:
Non-RIRs may provide bulk access to data, so why is it mandatory for servers to implement the NRO RDAP profile?

Reply:
Agreed that there could be actors that don't adhere to the NRO RDAP Profile which only the RIRs and their national and local registries are expected to. Perhaps this superseding should be made conditional in case one happens to be an NRO RDAP Profile-adhering actor, no?

Reply:
Thanks, sounds good.